_posts/release/2018-02-16-accumulo-1.7.4.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Apache Accumulo 1.7.4
+draft: true
+---
+
+Apache Accumulo 1.7.4 is a maintenance release on the 1.7 version branch. This release contains changes from 46 issues, comprised of bug-fixes, 
+performance improvements, build quality improvements, and more. See [Jira][JIRA_174] for a complete list.
+
+Below are resources for this release:
+
+* [User Manual][user_manual] - In-depth developer and administrator documentation.
+* [Javadocs][javadoc] - Accumulo 1.7.4 API
+* [Examples][examples] - Code with corresponding readme files that give step by step instructions for running example code.
+
+Accumulo follows [Semantic Versioning][semver] [guidelines][guidelines].  This release is a  “[patch version][patch_version]”, which means that
+only backwards compatible bug fixes are introduced in this version. A bug fix is defined as an internal change that fixes incorrect behavior. Users
+of any previous 1.7.x release are strongly encouraged to update as soon as possible to benefit from the  bug fixes with very little concern in change
+of underlying functionality.  As always, the Accumulo developers take API compatibility very seriously and have invested much time to ensure that we
+meet the promises set forth to our users. Users of 1.6 or earlier that are seeking to upgrade to 1.7 should consider 1.7.4 as a starting point.
+
+## Major Changes
+
+## Other Notable Changes
+
+* [ACCUMULO-4817] - Update build plugins and parent POM
+* [ACCUMULO-4809] - Avoid blocking during session clean up
+* [ACCUMULO-4787] - Close input stream in AccumuloReplicaSystem
+* [ACCUMULO-4778] - Cache table name to id map
+* [ACCUMULO-4777] - Root tablet got spammed with 1.8 million log entries
+* [ACCUMULO-4686] - 1.8 upgrade path does not update version on multiple volumes
+* [ACCUMULO-4676] - Use HTTPOnly flags in monitor
+* [ACCUMULO-4669] - RFile can create very large blocks when key statistics are not uniform
+* [ACCUMULO-4665] - Must use the "real" user for RPCs when Kerberos is enabled
+* [ACCUMULO-4633] - ArithmeticException on TabletServerBatchWriter#logStats
+* [ACCUMULO-4619] - Add splits hung forever
+
+## Upgrading
+
+View the [Upgrading Accumulo documentation][upgrade] for guidance.
+
+## Testing
+
+[JIRA_174]: https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12312121&version=12339577
+[upgrade]: https://accumulo.apache.org/docs/2.0/administration/upgrading
+[user_manual]: https://accumulo.apache.org/1.7/accumulo_user_manual.html
+[javadoc]: https://accumulo.apache.org/1.7/apidocs/
+[examples]: https://accumulo.apache.org/1.7/examples/
+[semver]: http://semver.org/
+[guidelines]: https://github.com/apache/accumulo/blob/rel/1.7.3/README.md#api
```

Comment:
Could update this for the 1.7.4 README which should eventually exist

Reply:
Fixed in 9f9ad9e13d